examples/BUCK

```diff
@@ -3,9 +3,6 @@ cxx_binary(
   srcs = [
     'main.cpp',
   ],
-  compiler_flags = [
```

Comment:
are you sure about this? ctti requires C++14.

BTW, how does buck handle different compilers + compiler args? Note ctti supports MSVC in addition to clang and gcc. The buck script must work on all supported platforms to be accepted.

Reply:
I can confirm that the Buck script works on Windows.